### ISSUE TITLE
Arraystacking in getChat Function

### ISSUE THREAD

**Issue by moelrobi** [AminoJS/Amino.JS#34]
**Describe the bug**
The Array when getChat is Called is stacking / filling up without any clearing. So with every call the Array Fills up.

**To Reproduce**
```js
//This Snippet shows the Bug in action!
    let chats = await Amino.getJoinedChats(community);
    chats.threads.forEach(async (el) => {
        let chat = await Amino.getChat(community, el.threadId);
        console.log(chat);
    })
```

**Expected behavior**
It should return one Messages Array per Call and clear it right after, instead it filles the Array endllessly.

**Desktop (please complete the following information):**
 - OS: Linux AsusLap 4.15.0-24-generic #26~16.04.1-Ubuntu SMP Fri Jun 15 14:35:08 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
 - Node-Version: v8.11.3
 - Version: master / 2.0.0

**Comment by felixfong227** [comment]
Can we like start the cleaning job once the array index reached 100(further discuss), and maybe add a new child method for `getChat()` for keep tracking how many object is inside the array and a method for the actual cleaning job

Examples
```javascript
const chats = await Amino.getJoinedChats(community);
chats.threads.forEach(async (el) => {
    const chat = await Amino.getChat(community, el.threadId);
    const howMany = await chat.stacks(); // #=> 10
    // If the stacks is over 20, then clean it up plz
    if(howMany >= 20){
        await chat.cleanUpStacks();
    }
});
```

**Comment by moelrobi** [comment]
Getting fixed in #39 

**Closed!**
